pkg/apiserver/rest/usecase/addon.go

```diff
@@ -204,6 +204,8 @@ func (u *defaultAddonHandler) StatusAddon(ctx context.Context, name string) (*ap
 	if err != nil && !errors2.IsNotFound(err) {
```

Comment:
Please change the code style, CI cannot pass.

Reply:
> Please change the code style, CI cannot pass.

updated